service/features/controller_create_delete_snapshot.feature

```diff
@@ -8,44 +8,41 @@ Feature: Isilon CSI interface
     Given a Isilon service
     When I call Probe
-    And I call CreateSnapshot "volume2=_=_=19=_=_=System=_=_=cluster2" "create_snapshot_name" "/ifs/data/csi-isilon"
+    And I call CreateSnapshot "volume2" "volume2=_=_=19=_=_=System=_=_=cluster2" "create_snapshot_name"
     Then the error contains "failed to get cluster config details for clusterName: 'cluster2'"
 
   Scenario: Create snapshot with internal server error
     Given a Isilon service
     When I call Probe
     And I induce error "CreateSnapshotError"
-    And I call CreateSnapshot "volume2=_=_=19=_=_=System" "create_snapshot_name" "/ifs/data/csi-isilon"
+    And I call CreateSnapshot "volume2" "volume2=_=_=19=_=_=System" "create_snapshot_name"
     Then the error contains "EOF"
 
   Scenario Outline: Create snapshot with negative or idempotent arguments
     Given a Isilon service
-    When I call CreateSnapshot <volumeID> <snapshotName> <isiPath>
+    When I call CreateSnapshot <volName> <volumeID> <snapshotName>
     Then the error contains <errormsg>
 
     Examples:
-    | volumeID                       | snapshotName                                  | isiPath                    | errormsg                               |
-    | "volume1=_=_=10=_=_=System"    | "create_snapshot_name"                        | "/ifs/data/csi-isilon"     | "source volume id is invalid"          |
-    | "volume2=_=_=19=_=_=System"    | "existent_snapshot_name"                      | "/ifs/data/csi-isilon"     | "already exists but is incompatible"   |
-    | "volume2=_=_=19=_=_=System"    | "existent_comp_snapshot_name"                 | "/ifs/data/csi-isilon"     | "none"                                 |
-    | "volume2=_=_=19=_=_=System"    | "existent_comp_snapshot_name"                 | "/ifs/data/csi-isilon"     | "none"                                 |
-    | "volume2=_=_=19=_=_=System"    | "existent_comp_snapshot_name_longer_than_max" | "/ifs/data/csi-isilon"     | "already exists but is incompatible"   |
-    | "volume2=_=_=19"               | "existent_comp_snapshot_name"                 | "/ifs/data/csi-isilon"     | "cannot be split into tokens"            |
-    | "volume2=_=_=19=_=_=System"    | "create_snapshot_name"                        | ""                         | "none"                                 |
-    | "volume2=_=_=19=_=_=System"    | "create_snapshot_name"                        | "none"                     | "none"                                 |
-    | "volume2=_=_=19=_=_=System"    | ""                                            | "/ifs/data/csi-isilon"     | "name cannot be empty"                 |
+    | volName        | volumeID                       | snapshotName                                  | errormsg                             |
+    | "volume1"      | "volume1=_=_=10=_=_=System"    | "create_snapshot_name"                        | "source volume id is invalid"        |

```

Comment:
Where is the error message coming from? Our code or external? id -> ID.

Reply:
Yes, it is an internal error message